tests/test_tgis_backend.py

```diff
@@ -65,10 +65,7 @@ def mock_tgis_proc_fail():
 
 def test_tgis_backend_is_registered():
     """Make sure that the TGIS backend is correctly registered with caikit"""
-    assert (
-        TGISBackend.backend_type
-        in caikit.core.module_backends.backend_types.MODULE_BACKEND_TYPES
-    )
+    assert hasattr(caikit.core.module_backends.backend_types, TGISBackend.backend_type)
```

Comment:
I'm like... reasonably certain that many prior versions of caikit supported this, based on the overridden `__getattr__` on `caikit.core.module_backends.backend_types`